common/static/coffee/src/discussion/views/new_post_view.coffee

```diff
@@ -87,7 +87,6 @@ if Backbone?
               url: url
               type: "POST"
               dataType: 'json'
-              async: false # TODO when the rest of the stuff below is made to work properly..
```

Comment:
This scares me a bit. So deleting this line means that the ajax request will be asynchronous? Can you see who wrote the TODO so we can ask for more context?